charts/supabase/templates/db/deployment.yaml

```diff
@@ -9,6 +9,8 @@ spec:
   {{- if not .Values.db.autoscaling.enabled }}
   replicas: {{ .Values.db.replicaCount }}
   {{- end }}
+  strategy:
+    type: Recreate
```

Comment:
So its not possible to have zero-downtime deployments? Is there no way we can make this happen? 😗

Reply:
As far as I'm aware no Database System allows Upgrading the datastructure while accepting connections, which will result in a downtime in case of an Upgrade to the Datebase.
If this is set to Rolling, the second DB Pod might not start or the Upgrade might result in Inconsistent Data